service/src/main/java/com/myhome/controllers/UserController.java

```diff
@@ -81,4 +90,34 @@
         .map(response -> ResponseEntity.status(HttpStatus.OK).body(response))
         .orElseGet(() -> ResponseEntity.status(HttpStatus.NOT_FOUND).build());
   }
+
+  @Operation(
+      description = "Request reset user password",
+      responses = {
+          @ApiResponse(
+              responseCode = "200",
+              description = "If password reset request accepted"
+          ),
+          @ApiResponse(
+              responseCode = "400",
+              description = "If password reset request rejected"
+          )
+      })
+  @PostMapping(path = "/user/password",
```

Comment:
API should be specified in api.yaml from now. This controller is already migrated :)